src/Psy/Configuration.php

```diff
@@ -772,6 +775,34 @@ public function getLoop()
     }
 
     /**
+     * Retrieves the tab completion status
+     *
+     * @return bool
+     */
+    public function getTabCompletion()
+    {
+        return function_exists('readline_info');
+    }
+
+    /**
+     *
+     */
+    public function getAutoCompleter()
+    {
+        $completer = new AutoCompleter();
```

Comment:
this should store the completer instance on `$this->completer` like config does with all other services.